From 9d844ce3f8e411bfb4237372b1a6d4aec21a0a65 Mon Sep 17 00:00:00 2001
From: Marco Biscaro <marcobiscaro2112@gmail.com>
Date: Sun, 26 Aug 2012 21:32:41 -0300
Subject: [PATCH] fdo#47157 adding Keywords to .desktop files
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adding Keywords to .desktop files, following the suggestions in
https://bugs.freedesktop.org/show_bug.cgi?id=47157#c0

Change-Id: Ic2a1428aaa12bc31b0af8488d166f78d5a970a1d
Reviewed-on: https://gerrit.libreoffice.org/490
Reviewed-by: Björn Michaelsen <bjoern.michaelsen@canonical.com>
Tested-by: Björn Michaelsen <bjoern.michaelsen@canonical.com>
---
 sysui/desktop/menus/base.desktop    | 1 +
 sysui/desktop/menus/calc.desktop    | 1 +
 sysui/desktop/menus/draw.desktop    | 1 +
 sysui/desktop/menus/impress.desktop | 1 +
 sysui/desktop/menus/math.desktop    | 1 +
 sysui/desktop/menus/writer.desktop  | 1 +
 6 files changed, 6 insertions(+)

diff --git a/sysui/desktop/menus/base.desktop b/sysui/desktop/menus/base.desktop
index 72cf467..578e5c7 100755
--- a/sysui/desktop/menus/base.desktop
+++ b/sysui/desktop/menus/base.desktop
@@ -9,6 +9,7 @@ MimeType=application/vnd.oasis.opendocument.database;application/vnd.sun.xml.bas
 Name=%PRODUCTNAME Base
 GenericName=Database Development
 Comment=Manage databases, create queries and reports to track and manage your information by using Base.
+Keywords=Data;SQL;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
diff --git a/sysui/desktop/menus/calc.desktop b/sysui/desktop/menus/calc.desktop
index e9f3250..638b9e2 100755
--- a/sysui/desktop/menus/calc.desktop
+++ b/sysui/desktop/menus/calc.desktop
@@ -9,6 +9,7 @@ MimeType=application/vnd.oasis.opendocument.spreadsheet;application/vnd.oasis.op
 Name=%PRODUCTNAME Calc
 GenericName=Spreadsheet
 Comment=Perform calculations, analyze information and manage lists in spreadsheets by using Calc.
+Keywords=Accounting;Stats;Chart;ods;xls;xlsx;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
diff --git a/sysui/desktop/menus/draw.desktop b/sysui/desktop/menus/draw.desktop
index 9511fd5..6d92845 100755
--- a/sysui/desktop/menus/draw.desktop
+++ b/sysui/desktop/menus/draw.desktop
@@ -9,6 +9,7 @@ MimeType=application/vnd.oasis.opendocument.graphics;application/vnd.oasis.opend
 Name=%PRODUCTNAME Draw
 GenericName=Drawing Program
 Comment=Create and edit drawings, flow charts, and logos by using Draw.
+Keywords=Vector;Schema;Diagram;odg;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
diff --git a/sysui/desktop/menus/impress.desktop b/sysui/desktop/menus/impress.desktop
index 2fb8650..d8ca703 100755
--- a/sysui/desktop/menus/impress.desktop
+++ b/sysui/desktop/menus/impress.desktop
@@ -9,6 +9,7 @@ MimeType=application/vnd.oasis.opendocument.presentation;application/vnd.oasis.o
 Name=%PRODUCTNAME Impress
 GenericName=Presentation
 Comment=Create and edit presentations for slideshows, meeting and Web pages by using Impress.
+Keywords=Slideshow;odp;ppt;pptx;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
diff --git a/sysui/desktop/menus/math.desktop b/sysui/desktop/menus/math.desktop
index 867711d..4381a49 100755
--- a/sysui/desktop/menus/math.desktop
+++ b/sysui/desktop/menus/math.desktop
@@ -10,6 +10,7 @@ MimeType=application/vnd.oasis.opendocument.formula;application/vnd.sun.xml.math
 Name=%PRODUCTNAME Math
 GenericName=Formula Editor
 Comment=Create and edit scientific formulas and equations by using Math.
+Keywords=Equation;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
diff --git a/sysui/desktop/menus/writer.desktop b/sysui/desktop/menus/writer.desktop
index 888b382..f76ac4e 100755
--- a/sysui/desktop/menus/writer.desktop
+++ b/sysui/desktop/menus/writer.desktop
@@ -9,6 +9,7 @@ MimeType=application/vnd.oasis.opendocument.text;application/vnd.oasis.opendocum
 Name=%PRODUCTNAME Writer
 GenericName=Word Processor
 Comment=Create and edit text and graphics in letters, reports, documents and Web pages by using Writer.
+Keywords=Text;Letter;CV;odt;doc;docx;
 InitialPreference=5
 X-Ayatana-Desktop-Shortcuts=X-New
 [X-New Shortcut Group]
-- 
1.7.11.7

